generators/server/templates/src/main/java/package/config/_SecurityConfiguration.java

```diff
@@ -71,7 +72,11 @@
 import org.zalando.problem.spring.web.advice.security.SecurityProblemSupport;
 
 @Configuration
+<%_ if (authenticationType === 'oauth2') { _%>
+@EnableOAuth2Sso
```

Comment:
use EnableOauth2Sso for now, until the migration to spring-security-5-oidc

Reply:
@mraible said it was better to stick with the older oauth2 implementation, so at the moment I would stick to this (besides, that's less work!)